firestore/src/jni/arena_ref.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIREBASE_FIRESTORE_SRC_JNI_ARENA_REF_H_
+#define FIREBASE_FIRESTORE_SRC_JNI_ARENA_REF_H_
+
+#include <jni.h>
+
+#include <cstdlib>
+
+#include "firestore/src/jni/jni_fwd.h"
+#include "firestore/src/jni/object.h"
+#include "firestore/src/jni/ownership.h"
+
+namespace firebase {
+namespace firestore {
+namespace jni {
+
+namespace {
+constexpr int64_t kInvalidKey = -1;
```

Comment:
Please move `kInvalidKey` to inside the `ArenaRef` class. It should probably be a private static variable of `ArenaRef`. Otherwise, it will be part of the `firebase::firestore::jni` namespace, which doesn't really make sense because it only applies to `ArenaRef`.